### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#168] [Esercizi - Viaggi/Piazza]
![Es 5](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/fd964aa7-64e9-4394-9b65-6a06f4d0b7d7)
- Verifica della somma in #18

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/1c6b7733-76e2-4102-820f-70c33114b342)

**Comment by Elia-Belli** [comment]
Abbiamo definito $\mathbb{Q}$ come insieme quoziente $\mathbb{Z} \times (\mathbb{Z} \setminus \\{0\\})/\rho$, dove $\rho$ è una relazione d'equivalenza tale che $(a,b)\rho(c,d)$ se $ad=bc$.

- Sia $\cdot := [(a,b)]\cdot[(a',b')]= [(a\cdot a',b\cdot b')]$ dobbiamo verificare che l'operazione è ben posta, o ben definita, ovvero che non dipende dalla scelta dei rappresentanti delle classi.
Dimostriamo quindi $(a,b) \rho (c,d) \land (a',b') \rho (c',d') \Rightarrow (a\cdot a',b\cdot b')\rho(c\cdot c',d\cdot d')$
 Per definizione di $\rho\ $: $ad = bc \land a'd'=b'c' \Rightarrow (aa')(dd')=(bb')(cc')$
 Sfruttando la commutatività e l'associatività del prodotto in $\mathbb{Z}$, possiamo scrivere $(ad)(a'd')=(bc)(b'c')$
 Usando l'ipotesi sostituiamo i termini $a'd'$ e $bc$, ottenendo l'identità $(ad)(b'c')=(ad)(b'c')$. $\blacksquare$

- $(\mathbb{Z},+,\cdot)$ induce su $\mathbb{Q}$ le proprietà di anello commutativo unitario; se con $0:= [0,0]= [0,b]\ \forall b\neq 0$ el. neutro per $+$ e $1:= [(1,1)]=[a,a]\ \forall a\neq 0$ el. neutro per $\cdot\ $, ora basta mostrare l'esistenza dell'inverso moltiplicativo:
  $[(a,b)][(b,a)] = [(ab,ba)] = [(ab,ab)] = [(1,1)] = 1,\ \forall (a,b), a \neq 0$
allora $\mathbb{Q}$ è un campo.
